integration/kubernetes/runtimeclass_workloads/pod-init.yaml

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-init
+spec:
+  runtimeClassName: kata
+  initContainers:
```

Comment:
On the Pod's definition you may want to set `restartPolicy` to `never`, this way it won't try to restart the init container over and over in case there is a bug.

I found the information about `restartPolicy`of init containers here:
https://kubernetes.io/docs/concepts/workloads/pods/init-containers/

Reply:
@wainersm I added to the pods definition